tests/optimize/test_lie_algebra.py

```diff
@@ -189,14 +189,13 @@ def lie_circuit():
     hamiltonian_np = qml.utils.sparse_hamiltonian(hamiltonian, wires).toarray()
     lie_algebra_np = hamiltonian_np @ rho - rho @ hamiltonian_np
 
-    phi_exact = expm(-0.001 * lie_algebra_np) @ phi
+    phi_exact = expm(-0.1 * lie_algebra_np * 2**nqubits) @ phi
```

Comment:
I'm curious how come this changed?

Reply:
So I discovered that this factor 2**N was missing, which did not affect this test due to the small stepsize.